src/Pact/Server/API.hs

```diff
@@ -146,120 +149,218 @@ writeSwagger :: FilePath -> Swagger -> IO ()
 writeSwagger fn = BSL8.writeFile fn . encode
 
 
+------------------------------------------------------------------
+------------------------------------------------------------------
 
--- ORPHANS for swagger
 
-instance ToSchema (Command Text)
+-- | Swagger ORPHANS
 
-instance ToSchema (CommandResult Hash)
+instance ToSchema SubmitBatch where
+  declareNamedSchema = lensyDeclareNamedSchema 3
 
-instance ToSchema SubmitBatch
+instance ToSchema (Command Text) where
+  declareNamedSchema =
+    (swaggerDescription "transaction command with stringified JSON payload (cmd)") .
+    (genericDeclareNamedSchema $
+      optionsOf $ optFieldLabel $
+                  replaceOrModify
+                  ("_cmdPayload","cmd")
+                  (lensyConstructorToNiceJson 4)
+    )
```

Comment:
Do we only have a `Text` instance for `ToSchema`, or can we have `instance ToSchema a => ToSchema (Comman a)`? If so we can eliminate the need for `-XFlexibleInstances`